app/src/main/java/edu/cascadia/mobas/pollinatorpathway/ui/login/PlantingDao.java

```diff
@@ -0,0 +1,32 @@
+package edu.cascadia.mobas.pollinatorpathway.ui.login;
+
+import androidx.room.Dao;
+import androidx.room.Delete;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+import androidx.room.Update;
+import java.util.List;
+
+
+@Dao
+public interface PlantingDao {
+
+    @Query("SELECT * FROM Planting WHERE plantingId = :plantingId")
+    public Planting getPlanting(long plantingId);
+
+    @Query("SELECT * FROM Planting WHERE name = :name")
+    public Planting getPlantingByText(String name);
+
+    @Query("SELECT * FROM Planting ORDER BY name ASC")
+    List<Planting> getAllPlantings();
+
```

Comment:
You will likely want to add a `List<Planting> getProfilePlantings(long userId)` method so that you can display all of the plantings associated with a profile on the profile page